### PR TITLE
Move to Quarkus runtime

### PR DESCRIPTION
**Pull request by jurakp**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,48 +1,50 @@
...
+# copy the custom cache config file into the keycloak conf dir
+RUN ["/bin/bash", "-c", "cp -Rv $INSTALL_FOLDER/conf/cache-ispn-jdbc-ping.xml $KC_FOLDER/conf/"]
+
 # copy custom modules
-RUN ["/bin/bash", "-c", "cp -Rv $INSTALL_FOLDER/modules/* $JBOSS_HOME/modules/"]
+RUN ["/bin/bash", "-c", "cp -Rv $INSTALL_FOLDER/modules/* $KC_FOLDER/providers/"]
 
 # copy welcome page
-RUN ["/bin/bash", "-c", "cp -Rv $INSTALL_FOLDER/welcome-content/* $JBOSS_HOME/welcome-content/"]
+#RUN ["/bin/bash", "-c", "cp -Rv $INSTALL_FOLDER/welcome-content/* $KC_FOLDER/welcome-content/"]
```
Comment (krisrjohnson21) [inline Dockerfile]: Question: Is this line commented out intentionally?

**Review by krisrjohnson21 — COMMENTED** [review]
One question from me! Would appreciate a second set of eyes from @paulswartz or @ianwestcott

File: Dockerfile (hunk 1 above)
Comment (jurakp) [reply Dockerfile]: Yes, it is not needed, I removed it.

**Review by paulswartz — APPROVED** [review]

**Review by krisrjohnson21 — APPROVED** [review]
👍

**Pull request merged**